### DISCUSSION TITLE
Adding a settings RC file for different developers to gitignore locally

### DISCUSSION THREAD

**Discussion by Kayrah87** [permafrost-dev/vue-ray#121] [Ideas]
I work in a team of developers. Some of use love Ray, and some of us hate it. I personally love it and want to use it to investigate everything rather than trawling through the console.

However, we now have a war in the commits. other developers will comment out the .use(RayPlugin) config and commit it. I will start working on a feature and have to go in and turn the damn thing back on again every single time.

Could this configuration be moved out to an env file, like a .rayrc.js file or something, in order for us to gitignore it, so I can have my local version with Ray always on, whilst other devs have theirs always off?

**Comment by patinthehat** [comment]
Hey @Kayrah87, sorry for the late response. You could use the `node-ray` feature `.disable()` by default and have your own code that calls `.enable()`.  This is the only solution I can think of off the top that might solve your issue. See the [node-ray](https://github.com/permafrost-dev/node-ray#reference) reference docs.

I imagine that everyone would have a `.rayrc.js` that's gitignored but included in the main script, and you'd enable it from there.
